### PR TITLE
Fixed responsiveness for footer component

### LINKED ISSUE #336: Website: Improve footer responsiveness
#### Description
Currently:
Screen Shot 2022-05-18 at 1 10 40 PM

#### Expected Behavior
Breakpoints and column sizes should be adjusted to better fit browser width responsively.

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by neeraj-2**
Signed-off-by: Neeraj Anand <neeraj.2@iitj.ac.in>

**Description**

This PR fixes #336. 
- Added some bootstrap inline styling for fixing the responsiveness of footer component

**Demo After Changes**

https://user-images.githubusercontent.com/56186425/186788761-f1993f42-6a0e-4f1a-88f5-f38a3bb73e64.mov

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by asubedy — CHANGES_REQUESTED** [review]
Let's center align the Service Mesh Performance Heading and make it a little bigger in size
image

**Comment by neeraj-2** [comment]
> Checking in... it has been awhile since we've heard from you on this issue. Are you still working on it? Please let us know and please don't hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
> 
> > Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a star [star on the project](../stargazers) smile on the project.

Actually, the minor exams for my 7th sem are going on. So, I will continue from the 4th of September. Thanks!

**Comment by neeraj-2** [comment]
> Let's center align the Service Mesh Performance Heading and make it a little bigger in size image

Thanks, @asubedy for your review and suggestion. I have pushed the changes. Sorry for being late. Actually, my exams were going on. Will be more regular from onwards. Thanks!

File: docs/_includes/footer.html [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
         <div class="row">
-          <div class="col-sm-3 padding-right-50"> <img class="foot-logo" src="/assets/img/logo/smp-light-text-side.png" height="100" width="230" alt="SMP" > </div>
-          <div class="col-sm-9">
-            <div class="pos-info">
-              <ul class="row">
-                <li class="col-sm-4"> <br>
-                  <h6 class="rspnsv">Service Mesh Performance</h6>
+          <div class="col-lg-4 col-md-10 mb-4"> <img class="foot-logo" src="/assets/img/logo/smp-light-text-side.png" height="100" width="230" alt="SMP" > </div>
+          <div class="col-lg-3 col-md-6 mb-4">
+            <h6 class="rspnsv" style="color: #fff; font-weight: 600; text-align: center; font-size: medium;">Service Mesh Performance</h6>
```
Comment (asubedy) [inline docs/_includes/footer.html:8]:
```suggestion
            <h6 class="rspnsv" style="color: #fff; font-weight: 600; text-align: center; font-size: larger;">Service Mesh Performance</h6>
```

**Review by asubedy — APPROVED** [review]

**Pull request merged**